.doc_gen/cross-content/cross_FMPlayground_NetV3_block.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE chapter PUBLIC "-//OASIS//DTD DocBook XML V4.5//EN" "file://zonbook/docbookx.dtd" [
+  <!ENTITY % phrases-shared SYSTEM "file://AWSShared/common/phrases-shared.ent">
+  %phrases-shared;
+]>
+<block>
+  <para>
+	  .NET FM Playground is a .NET MAUI Blazor sample application showcasing how to leverage &BRlong; from C# code. 
+	  This example application is not production-ready. It illustrates how .NET and C# developers can leverage &BR; 
+	  to build generative AI enabled applications. In order to let you test and interact with the foundation models accesible
+	  via &BR; API, the .NET FM Playground offers you four playgrounds:
+  </para>
+  <itemizedlist>
+    <listitem>
+      <para>A text playground</para>
+    </listitem>
+    <listitem>
+      <para>A chat playground</para>
+    </listitem>
+    <listitem>
+      <para>A voice chat playground</para>
+    </listitem>
+    <listitem>
+      <para>An image playground</para>
+    </listitem>
+  </itemizedlist>
+  <para>In addition, it also lists and displays the foundation models you have access to and their characteristics. For source code and deployment
+    instructions, see the project in <ulink url="https://github.com/build-on-aws/dotnet-fm-playground">GitHub</ulink>.
+  </para>
+</block>
```

Comment:
Add empty line (carriage-return from last line of code)
Required to parse correctly.

Reply:
I've applied the fix